openapi/p4pa-auth.openapi.yaml

```
@@ -233,6 +233,46 @@ paths:
           description: Conflict
         '429':
           description: Too Many Requests
+  /oauth/clients/searchByFilters:
```

what about just `search`?
```suggestion
  /oauth/clients/search:
```